sequencer/src/abci/application.rs

```diff
@@ -96,7 +96,7 @@ impl Application for StarknetApp {
         match tx.transaction_type {
             TransactionType::Declare { program: _ } => info!("Received declare transaction"),
             TransactionType::DeployAccount { .. } => info!("Received deploy transaction"),
-            TransactionType::Invoke => todo!(),
+            TransactionType::Invoke { .. } => info!("Received invoke transaction"),
```

Comment:
There could be a validation here that checks whether the smart contract exists in the state. The same should probably apply for the `DeployAccount` transaction but that could be done in a different PR.